src/views/auth/error.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'default.html' %}
+{% set title = 'Authentication Error' %}
+{% block pageHeading %}Oops!{% endblock %}
+{% block pageContent %}
+{% if errorCode === 'TOO_YOUNG' %}
+<p>Unfortunately, due to legal reasons, only people who are 18 or over on the day of the hackathon can attend :(</p>
```

Comment:
How do we feel about using emoji on our error page? Could use 🙁 or something similar... Up to you whether you feel that's in line with our current style or not!